src/components/map/MapService.js

```diff
@@ -1744,6 +1752,8 @@ goog.require('ga_urlutils_service');
                 layerTimestamps);
           }
 
+          gaTime.allowStatusUpdate = true;
```

Comment:
I think this activation should be done in the gaTime service itself, but that's probably hardly possible as we would face a race condition when we depend on the $q.all promise in both locations, right?